scripts/scale.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+export VALID_NAMESPACES=("hocs-gamma" "hocs-delta" "hocs-qax")
+
+current=$(kubectl config current-context)
+if [[ ${VALID_NAMESPACES[*]} =~ "$current" ]] ; then
+  echo "Unauthorised Namespace" >&2
+  exit 1
+fi
+
+if [ "$#" -ne 1 ]; then
+  echo "Usage: scale.sh down/up" >&2
+  exit 1
+fi
+if [ "$1" != "down" -a "$1" != "up" ]; then
```

Comment:
I would have the "happy path" `exit 0` and run the contents of this if-block at the bottom (without the `if`).

Reply:
Done.